src/python/gem5/components/boards/experimental/lupv_board.py

```diff
@@ -185,7 +185,9 @@ def _setup_board(self) -> None:
         # point for our bbl to use upon startup, and will
         # remain unused during the simulation
         self.pic.n_src = 0
-        self.pic.hart_config = ""
+        self.pic.hart_config = ",".join(
+            ["MS" for _ in range(self.processor.get_num_cores())]
```

Comment:
We should remove "S" (supervisor external interrupt) in the hart config to avoid conflict with `LupioPIC`

https://github.com/gem5/gem5/blob/9bd71bff0c046ecb69bf3a4fcc1bf51004b5cefa/src/python/gem5/components/boards/experimental/lupv_board.py#L451-L458

```suggestion
            ["M" for _ in range(self.processor.get_num_cores())]
```